themes/theme-b2b/src/components/DatePicker.styles.ts

```diff
@@ -0,0 +1,15 @@
+import { ThemeComponent } from '@marigold/system';
+import { cva } from 'class-variance-authority';
+
+export const DatePicker: ThemeComponent<'DatePicker'> = {
+  container: cva(),
+  button: cva([
+    ' h-full w-6',
+    'p-0',
```

Comment:
I would leave the padding, I think it looks better, what do you think?

Reply:
yeah sure and we don't need it now